packages/button/package.json

```diff
@@ -1,9 +1,20 @@
 {
   "name": "@constellation-components/button",
-  "author": "",
-  "private": "false",
-  "description": "",
-  "main": "dist/packages/button/button.js",
-  "version": "1.1.1",
-  "gitHead": "1c99d86ac55fd5dc8f367abd8132c0f423469391"
+  "author": "maecapozzi <maecapozzi@gmail.com>",
+  "version": "1.0.1-alpha.5",
+  "description": "> TODO: description",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "tsc -p tsconfig.json"
+  },
+  "homepage": "https://github.com/maecapozzi/constellation#readme",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
```

Comment:
I forgot to add this, which is why I was accidentally publishing them as private components.